src/app/settings/layout.tsx

```diff
@@ -0,0 +1,101 @@
+"use client";
+
+import { Avatar, AvatarFallback, AvatarImage } from "@/components/ui/avatar";
+import { Button } from "@/components/ui/button";
+import useMetadata from "@/lib/nostr/useMetadata";
+import useSession from "@/lib/nostr/useSession";
+import { cn } from "@/lib/utils";
+
+import { Bell, Brush, Cog, User, Server } from "lucide-react";
+import Link from "next/link";
+import { usePathname } from "next/navigation";
+
+// check if signed in, if not, redirect to sign in page
+
+const links = [
+  {
+    name: "Profile",
+    href: "/settings/profile",
+    Icon: User,
+  },
+  {
+    name: "Account",
+    href: "/settings/account",
+    Icon: Cog,
+  },
+  {
+    name: "Relays",
+    href: "/settings/relays",
+    Icon: Server,
+  },
+  {
+    name: "Appearance",
+    href: "/settings/appearance",
+    Icon: Brush,
+  },
+  {
+    name: "Notifications",
+    href: "/settings/notifications",
+    Icon: Bell,
+  },
+];
+
+export default function SettingsLayout({
+  children,
+}: {
+  children: React.ReactNode;
+}) {
+  const { picture, initials, name } = useSession();
+  const metadata = useMetadata();
+  const pathname = usePathname();
```

Comment:
I think I wanna load them on the server, not on the client.

maybe in a follow up PR if someone knows a better way